src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export function Person({ person }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+
+      {person.isMarried === true
+        ? (<p className="Person__partner">{`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}</p>)
+        : (<p className="Person__partner">I am not married</p>)}
+    </section>
```

Comment:
decide before jsx what text will be rendered and just render in jsx. something like that
```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```